DependencyInjection/CmfSeoExtension.php

```diff
@@ -79,9 +70,8 @@ public function load(array $configs, ContainerBuilder $container)
             $this->loadSonataAdmin($config['sonata_admin_extension'], $loader, $container, $sonataBundles);
         }
 
-        if ($config['enable_content_listener']) {
-
-            $loader->load('content-listener.xml');
+        if ($this->isConfigEnabled($container, $config['content_listener'])) {
+            $this->loadContentListener($config['content_listener'], $loader, $container);
 
             if ($this->isConfigEnabled($container, $config['alternate_locale'])) {
```

Comment:
@ElectricMaxxx does it make sense that this is only loaded when the content listener is enabled? could we not need alternate locale for other things too, like the sitemap?